### ISSUE TITLE
Bottleneck for sculk emitters

### ISSUE THREAD

**Issue by deadspidervenom** [bl4ckscor3/UnnamedSculkMod#8]
Issue 
-------
While playing with other mods, When trying to move a large amount of items from 1 storage to another(more then you would want to do manually), there is a severe bottleneck due to the speed of hoppers.

I would recommend changing it from a barrel to something like "sculk paste" you can put on faces of a block to let the emitter work on it.
+++

**Comment by bl4ckscor3** [comment]
FWIW, you can already modify the blocks a sculk emitter is able to extract out of with the `sculktransporting:sculk_emitter_can_extract_from` block tag.

**Comment by deadspidervenom** [comment]
> FWIW, you can already modify the blocks a sculk emitter is able to extract out of with the `sculktransporting:sculk_emitter_can_extract_from` block tag.

Is this for like making mods or can i do this via configs/commands?

**Comment by bl4ckscor3** [comment]
Using datapacks.
https://minecraft.wiki/w/Data_pack
https://minecraft.wiki/w/Tag

**Comment by deadspidervenom** [comment]
ahh, yea that beyond me

**Comment by deadspidervenom** [comment]
> Using datapacks. https://minecraft.wiki/w/Data_pack https://minecraft.wiki/w/Tag
 
 Would it be hard to make it a "whitelist" config option.
 
 Like "Custom Emitter storage blocks"

**Comment by bl4ckscor3** [comment]
No, but tags are the way to go for these kinds of things in my opinion.
